pkg/endpoint/policy.go

```diff
@@ -533,7 +612,8 @@ func (e *Endpoint) regeneratePolicy(owner Owner, opts models.ConfigurationMap) (
 			"policyRevision.repo": revision,
 		}).Debug("Skipping policy recalculation")
 		// This revision already computed, but may still need to be applied to BPF
-		return e.nextPolicyRevision > e.policyRevision, false, nil, nil, nil
+		// REVIEW: Why does this means the policy was changed?!
```

Comment:
Maybe the debug message isn't as clear as it should be? This check is only concerning label-based policy. I'd have to stare at the code longer to see why if the label maps are equal whether we can return here.

Reply:
Discussed offline with @tgraf , comments updated